test/e2e/scenario/transport.scenario.ts

```diff
@@ -0,0 +1,55 @@
+import { ExperimentalFeature } from '@datadog/browser-core'
+import { createTest, flushEvents } from '../lib/framework'
+import { getBrowserName, getPlatformName, withBrowserLogs } from '../lib/helpers/browser'
+
+describe('transport', () => {
+  describe('data compression', () => {
+    createTest('send RUM data compressed')
+      .withRum({
+        enableExperimentalFeatures: [ExperimentalFeature.COMPRESS_BATCH],
+      })
+      .run(async ({ intakeRegistry }) => {
+        await flushEvents()
+
+        expect(intakeRegistry.rumRequests.length).toBe(2)
+
+        const plainRequest = intakeRegistry.rumRequests.find((request) => request.encoding === null)
+        const deflateRequest = intakeRegistry.rumRequests.find((request) => request.encoding === 'deflate')
+
+        // The last view update should be sent without compression
+        expect(plainRequest!.events).toEqual([
+          jasmine.objectContaining({
+            type: 'view',
+          }),
+        ])
+
+        // Other data should be sent encoded
+        expect(deflateRequest!.events.length).toBeGreaterThan(0)
+      })
+
+    // Ignore this test on Safari desktop and Firefox because the Worker actually works even with
+    // CSP restriction.
+    // TODO: Remove this condition when upgrading to Safari 15 and Firefox 99
+    if (!((getBrowserName() === 'safari' && getPlatformName() === 'macos') || getBrowserName() === 'firefox')) {
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏did you consider bumping them?

Reply:
I did, but I decided that it was not the right place to do it, as it might involve more things (everywhere we use `getBrowserName` for skipping tests, we should see if it still makes sense with newer versions)